src/wsteth-exchange-rate-adapter/WstEthStEthExchangeRateChainlinkAdapter.sol

```diff
@@ -4,16 +4,22 @@ pragma solidity 0.8.21;
 import {IStEth} from "./interfaces/IStEth.sol";
 import {MinimalAggregatorV3Interface} from "./interfaces/MinimalAggregatorV3Interface.sol";
 
-/// @title WstEthEthExchangeRateChainlinkAdapter
+/// @title WstEthStEthExchangeRateChainlinkAdapter
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
-/// @notice wstETH/ETH exchange rate price feed.
-/// @dev This contract should only be used as price feed for `ChainlinkOracle`.
-contract WstEthEthExchangeRateChainlinkAdapter is MinimalAggregatorV3Interface {
+/// @notice wstETH/stETH exchange rate price feed.
+/// @dev This contract should only be used as price feed for `MorphoChainlinkOracleV2`.
+contract WstEthStEthExchangeRateChainlinkAdapter is MinimalAggregatorV3Interface {
+    /// @inheritdoc MinimalAggregatorV3Interface
     uint8 public constant decimals = 18;
-    string public constant description = "wstETH/ETH exchange rate";
+
+    /// @notice The description of the price feed.
+    string public constant description = "wstETH/stETH exchange rate";
+
+    /// @notice The address of stETH on Mainnet.
     IStEth public constant ST_ETH = IStEth(0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84);
 
+    /// @inheritdoc MinimalAggregatorV3Interface
```

Comment:
I think that "Only the `answer` field is used by `MorphoChainlinkOracleV2`." isn't clear enough on the fact that the other values returned here are non-sense. I would add:
```suggestion
    /// @inheritdoc MinimalAggregatorV3Interface
    /// @dev Returns zero for roundId, startedAt, updatedAt and answeredInRound.
```